modules/instance_template/variables.tf

```diff
@@ -14,6 +14,12 @@
  * limitations under the License.
  */
 
+variable "project_id" {
+  type        = string
+  description = "The GCP project ID"
+  default     = ""
```

Comment:
What will happen if this value is left as the default?

This property should probably have a `null` default and we should make sure the default works. Please update tests/examples so that some of them *don't* set this variable (and hence use the default).

Reply:
Thank you for the tip.
Updated default value in modules.
Ensured that variable `project_id` announced in all `examples` and `tests`.
Added symlink to `terraform.tfvars` for `mig_with_percent` tests.

Reply:
> Ensured that variable project_id announced in all examples and tests.

I specifically want some of the examples to *not* explicitly set the project_id and confirm how it behaves.

If a variable has a default value, we need to test that the module works with the default value.

Reply:
Terraform failed, in case `project` is not set (null)
![image](https://user-images.githubusercontent.com/15910313/63608072-bb751400-c5dc-11e9-8ff7-c7173a21badd.png)